### ISSUE TITLE
Show Active/Visited State

### ISSUE THREAD

**Issue by johnchourajr** [dmfrancisco/JS.coach#255]
Just a few usability/design considerations for the list view.

I'd love to see an active state for the list view.

![image](https://cloud.githubusercontent.com/assets/6431920/19053275/05f23baa-896f-11e6-8e03-ed364c1c9123.png)

If you add a `background-color: rgba(0,0,0,.05);` to the `<a/>` link, it should do the trick.

---

Also if there were an `opacity: .5;` on visited listings.

---

That is all :)

**Comment by dmfrancisco** [comment]
Yes, makes total sense 👍  I just added to the [backlog](https://github.com/dmfrancisco/JS.coach/projects/1)!
